sdk/eventhubs/azure-messaging-eventhubs/CMakeLists.txt

```diff
@@ -114,8 +114,10 @@ target_link_libraries(azure-messaging-eventhubs PUBLIC Azure::azure-core Azure::
 target_compile_definitions(azure-messaging-eventhubs PRIVATE _azure_BUILDING_SDK)
 
 if(USE_RUST_AMQP)
+  message(STATUS "Using Rust AMQP")
   target_compile_definitions(azure-messaging-eventhubs PRIVATE ENABLE_RUST_AMQP)
 else()
+  message(STATUS "Using uAMQP")
```

Comment:
Should we put recommendations in here to use the rust implementation. I fully expect we will deprecate the uAMQP adapter.

Reply:
Using uAMQP is the fallback right now - if we can't use Rust (for whatever reason - currently macos and 32bit for example), we fall back to uAMQP.

This is here because the shape of the AMQP API surface changes slightly with Rust vs uAMQP (there are a bunch of callback functions in the uAMQP surface that aren't present in the Rust API surface).